### PR TITLE
KER-207: Added cookie management page

### PR DESCRIPTION
**Pull request by arpargo**
A way to manage cookies after modal is no longer shown as required.

### CODE REVIEW

File: src/components/CookieBar/CookieBar.js [hunk 1]
```diff
@@ -1,37 +1,15 @@
...
-        // called when consents are saved
-        // handle changes like:
-        if (consents.matomo) {
-          enableMatomoTracking();
-        }
-      },
-    };
+    const config = getHDSCookieConfig(siteName, language, setLanguage);
+    return config;
+    // return getHDSCookieConfig(siteName, language, setLanguage);
```
Comment (mikkojamG) [inline src/components/CookieBar/CookieBar.js:12]: Could return `getHDSCookieConfig(siteName, language, setLanguage);` straight away. Also commented code could be removed

**Review by mikkojamG — COMMENTED** [review]
Works nicely 👍  Left one minor comment 🙂

File: src/components/CookieBar/CookieBar.js (hunk 1 above)
Comment (arpargo) [reply src/components/CookieBar/CookieBar.js:12]: For unknown reason when trying to return it directly it didn't work... :)

Comment (mikkojamG) [reply src/components/CookieBar/CookieBar.js:12]: Lol, well the commented code could still be removed 😁

Comment (arpargo) [reply src/components/CookieBar/CookieBar.js]: That has now been done and committed.

**Review by mikkojamG — APPROVED** [review]

**Pull request merged**